app/models/categorization.rb

```diff
@@ -0,0 +1,6 @@
+class Categorization < ApplicationRecord
+	belongs_to :restaurant
+	belongs_to :category
+	validates :restaurant_id, presence: true
+	validates :category_id, presence: true
+end
```

Comment:
1) strange indentation

2) In rails 5, you do not need these validations. They are automatic with a `belongs_to` relationship. Please remove